### ISSUE TITLE
Grid Searching with SVMRbf using 'leave-one-out' through error

### ISSUE THREAD

**Issue by sumeetssaurav** [Gijom/TEAP#7]
Hello,

I am trying to run the train_and_test_deap file with the following setting:

For Classification

parameters.verbose = false;

parameters.normalize = 3;
parameters.nbClasses = 2;
parameters.lower_limit = -0.5;
parameters.upper_limit = 0.5;
parameters.grid_search = true;
parameters.featSelection = 'Fisher';
parameters.classifier = 'SVMRbf';
parameters.is_fusion = false;
parameters.cross_validation = 'leave-one-out';

and for Regression

parameters.normalize = 3;
%regression
parameters.nbClasses = -1;
parameters.lower_limit = -0.5;
parameters.upper_limit = 0.5;
parameters.grid_search = true;
parameters.featSelection = 'none';
parameters.classifier = 'SVR_RBF';
parameters.cross_validation = 'leave-one-out';

on running the code its showing 'Index Exceed Matrix Dimension' error.

Please help me out.
![untitled](https://cloud.githubusercontent.com/assets/13672595/24282824/ea349ca0-1087-11e7-9a8d-f25b74ffe765.png)

**Comment by msoley** [comment]
Thanks for checking the code. The code that is provided in test is to reproduce the results in the paper and not the main part of the Toolbox. Our main goal is to provice a feature extraction tool. 

I suspect what is happening here is that the training set does not have all the labels possible. I tried to fix it pull the last version and it should be working. In any case, it does not make that much sense to train a model when all the labels are not present so you might want to fix it in the cross-validation code. The machine learning code was written a while ago and Matlab machine learning capabilities have advanced so you might want to try to do that part yoursefl and take advantage of the new featuers. 

I appreciate that you report the bugs though.
